app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewHolder.kt

```diff
@@ -135,10 +117,17 @@ class AttendeeViewHolder(private val binding: ItemAttendeeBinding) : RecyclerVie
         }
         requiredList.clear()
         customForm.forEach { form ->
-            setupCustomFormWithFields(form, textWatcher)
+            setupCustomFormWithFields(form, textWatcher, attendee, ticket, eventId, position)
         }
     }
-    private fun setupCustomFormWithFields(form: CustomForm, textWatcher: TextWatcher) {
+    private fun setupCustomFormWithFields(
+        form: CustomForm,
+        textWatcher: TextWatcher,
+        attendee: Attendee,
+        ticket: Ticket,
+        eventId: Long,
+        position: Int
```

Comment:
These are a lot of parameters, why weren't they required previously, why are they required now?

Reply:
Previously, it is only about setting up EditText view, for now, it is Spinner so I need to add a few more parameters just to set up listener, save gender option on screen rotation,... Let me see if I can better this while resolving conflicts